AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/bookings-list/bookings-list.component.ts

```diff
@@ -146,11 +150,12 @@ export class BookingsListComponent implements OnInit, OnDestroy {
         this.loaded = this.error = false;
         const searchTerm = this.bookingPersistService.searchTerm || '';
         const venueIds = this.bookingPersistService.selectedVenueIds;
+        const CaseTypes = this.bookingPersistService.selectedCaseTypes;
```

Comment:
sorry to nitpick - could we name this `caseTypes` (ie camelCase) for consistency?